### PR TITLE
add notion of a banner, which can be used as an instance description

### PR DESCRIPTION
**Pull request by trevj**
It's another switch for `run_cloud.sh` which, when set, will appear as the instance's description in the uProxy contacts list.

### CODE REVIEW

File: sshd/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,10 @@ RUN adduser --disabled-password --gecos 'uProxy Giver' --ingroup giver giver
 RUN addgroup getter
 RUN adduser --disabled-password --gecos 'uProxy Getter' --ingroup getter getter
 
+COPY banner /
+RUN chmod 644 /banner
+RUN chown giver: /banner
```
Comment (lucyhe) [inline sshd/Dockerfile]: What does the ":" mean?

**Comment by lucyhe** [comment]
Just a question, but :+1: !

File: sshd/Dockerfile (hunk 1 above)
Comment (trevj) [inline sshd/Dockerfile]: The notation is `user:group` but if you omit the group then the user's *primary* group will be used.

Comment (lucyhe) [inline sshd/Dockerfile]: Cool! Thanks!

**Pull request merged**